dist/sprestlib.d.ts

```diff
@@ -18,22 +18,20 @@ declare namespace sprLib {
   interface FileInfoOptions {
     version?: number;
   }
-  class file {
```

Comment:
Removed these classes. Added **IFile** interface and **file** function file to replace the 'file' class. This was repeated in the other classes.